types/shared.d.ts

```diff
@@ -473,4 +473,5 @@ export type ConnectElementTagName =
   | "balances"
   | "payouts-list"
   | "app-install"
-  | "app-viewport";
+  | "app-viewport"
```

Comment:
This PR is missing all of the setters related to this component

Reply:
Added the setters in `types/config.ts`